lib/guardian_phoenix.ex

```diff
@@ -1,18 +0,0 @@
-defmodule GuardianPhoenix do
```

Comment:
I'm not sure we want to delete this module, what was the rationale there?

Reply:
That you do not need that module.